engine.js

```diff
@@ -295,11 +357,11 @@ var hmdajson = require('./lib/hmdajson'),
         }
 
         if (rule.hasOwnProperty('if')) {
```

Comment:
Good call switching this to ternary. I felt something was wrong with returning in the if here when I wrote it but didn't figure it out.